addons/rating/controllers/main.py

```diff
@@ -46,11 +46,9 @@ def action_open_rating(self, token, rate, **kwargs):
             'rate': rate,
         })
 
-    @http.route(['/rate/<string:token>/submit_feedback'], type="http", auth="public", methods=['post'], website=True)
-    def action_submit_rating(self, token, rate, **kwargs):
-        rate = int(rate)
-        if rate not in (1, 3, 5):
-            raise ValueError(_("Incorrect rating: should be 1, 3 or 5 (received %d)"), rate)
+    @http.route(['/rate/<string:token>/submit_feedback'], type="http", auth="public", methods=['post', 'get'], website=True)
+    def action_submit_rating(self, token, rate=False, **kwargs):
```

Comment:
I just did the change, but i dont understand why it is needed. Is it because of the 'rate = int(rate) ' ? I thought 'False' would just be compute to 0, so it shouldn't be a problem.

Reply:
Actually we expect an integer for the rating, so 0 is better than False as a default value. Moreover 0 as rating somehow means "no rating" or "not rated yet", hence making more sense as a default behavior that is "display this page without really thinking of the rate value itself".

Reply:
ok thanks.